src/auth.md

```diff
@@ -63,7 +63,7 @@ website (such as "https://yourapp.example.com/oidc-callback") or mobile app
 
 Your application can use any OIDC client to negotiate the authentication flow
 but it must support OIDC authorization code flow with [PKCE][pkce]{:.external}.
-See the [Example Web Browser OIDC Consumer][oidcjs-example] section below for a
+See the [Example Web Browser OIDC Consumer](#oidcjs-example-anchor) section below for a
```

Comment:
This link is currently broken in the wild.

Reply:
I don't like the anchor suffix. It's redundant.

Reply:
Using a suffix that'll never show up in a title ensures we wont end up generating multiple IDs on the page. The thinking is that someone is unlikely to put the word 'anchor' in their heading, but it's likely that a heading may change without updating the dom.

I've gone and removed the `-anchor` suffix. Lets just try to keep the anchor IDs different to what we generate with titles and it should all be fine 👌

Reply:
We should auto generate anchors from section headings and the linter should detect broken links.

Reply:
+1, lets investigate a linter in a followup PR and remove this when we get there.